test/Models/RegionProviderListSpec.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+/*global require,describe,it,expect,beforeEach,fail*/
+
+var Terria = require('../../lib/Models/Terria');
+var RegionProviderList = require('../../lib/Models/RegionProviderList');
+var RegionProvider = require('../../lib/Models/RegionProvider');
+var DataTable = require('../../lib/Map/DataTable.js');
+var terria;
+var rpl;
+
+beforeEach(function() {
+    terria = new Terria({
+        baseUrl: './',
+        regionMappingDefinitionsUrl: 'test/csv/regionMapping.json',
+    });
+    terria.corsProxy.baseProxyUrl = ""; // there is no localhost:3002/proxy, so ...
```

Comment:
Is this necessary because we're calling `corsProxy.getURL` directly somewhere, so it's always trying to proxy even when it shouldn't?